README.md

```diff
@@ -55,19 +55,25 @@ Made with [Rust](https://www.rust-lang.org), [Actix](https://actix.rs/), [Infern
 - Open source, [AGPL License](/LICENSE).
 - Self hostable, easy to deploy.
   - Comes with [Docker](#docker), [Ansible](#ansible), [Kubernetes](#kubernetes).
+- Only a minimum of a username and password is required to sign up!
+- User avatar support.
 - Clean, mobile-friendly interface.
   - Live-updating Comment threads.
   - Full vote scores `(+/-)` like old reddit.
   - Themes, including light, dark, and solarized.
   - Emojis with autocomplete support. Start typing `:`
   - User tagging using `@`, Community tagging using `#`.
+  - Integrated image uploading in both posts and comments.
+  - A post can consist of a title and any comination of self text, a URL, or nothing else.
```

Comment:
combination

Reply:
Sorry, I'm not sure what you mean. Not familiar with Git/GitHub lingo yet.

Reply:
just saying you have a typo in there, `comination` ;)

Reply:
Oh, I'll fix that right now!